lib/screens/settings.dart

```diff
@@ -19,45 +21,78 @@ class _SettingsScreenState extends State<SettingsScreen> {
   Widget build(BuildContext context) {
     return Container(
         color: Colors.white,
-        padding: EdgeInsets.all(50),
-        child: Align(
-            alignment: Alignment.center,
-            child: Column(
+        padding: EdgeInsets.only(top: 48),
+        child: Column (
+            children: <Widget> [
+              Align (
+                alignment: Alignment.center,
+              child: Column (
               mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
               children: <Widget>[
                 ClipOval(
                     child: Image.network(imageUrl,
                         width: 100, height: 100, fit: BoxFit.cover)),
                 SizedBox(height: 20),
```

Comment:
I know this was my code by instead of a `SizedBox` it is better to add a padding in `ClipOval` and add `const EdgeInset.only(bottom: 20)`